### PR TITLE
Fix the fastNLP model train batch size.

### PR DESCRIPTION
**Pull request by xuzhao9**
In the documentation, the fastNLP Bert QA task uses batch size=6 for training. This PR fixes the correct train batch size.

Source:
https://fastnlp.readthedocs.io/zh/latest/tutorials/extend_1_bert_embedding.html

Original code:
`
trainer = Trainer(data_bundle.get_dataset('train'), model, loss=loss, optimizer=optimizer,
                  sampler=BucketSampler(seq_len_field_name='context_len'),
                  dev_data=data_bundle.get_dataset('dev'), metrics=metric,
                  callbacks=callbacks, device=device, batch_size=6, num_workers=2, n_epochs=2, print_every=1,
                  test_use_tqdm=False, update_every=10)
`

### CODE REVIEW

File: torchbenchmark/models/fastNLP_Bert/cmrc2018_simulator.py [hunk 1]
```diff
@@ -4,12 +4,10 @@
 """
 import os
 import pathlib
-import numpy
 import json
 import random
-import fastNLP
 
-TRAIN_BATCH_SIZE = 1
+TRAIN_BATCH_SIZE = 6
```
Comment (aaronenyeshi) [inline torchbenchmark/models/fastNLP_Bert/cmrc2018_simulator.py:10]: It may be good to leave a comment here to claim that this number comes from the documentation/src code.

**Review by aaronenyeshi — APPROVED** [review]
LGTM, see comment that we might want to add our reason in code too.

File: torchbenchmark/models/fastNLP_Bert/cmrc2018_simulator.py (hunk 1 above)
Comment (xuzhao9) [reply torchbenchmark/models/fastNLP_Bert/cmrc2018_simulator.py:12]: Thanks! This is resolved.

**Review by aaronenyeshi — APPROVED** [review]
LGTM, please import latest to phab
